app/src/main/java/com/example/arc_exapmle/NoteUI.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.arc_exapmle
+
+class NoteUI(noteEntity: NoteEntity?) {
+    internal var title: String? = null
+    internal var description: String? = null
+    internal var priority: Int? = null
```

Comment:
chera internal gozashti?
chera nullable hastan?

Reply:
In ro ham dorost kardam va taghiresh dadam